apps/condo/domains/ticket/components/TicketHint/FormTicketHintCard.tsx

```diff
@@ -0,0 +1,54 @@
+import { Col, Row, Typography } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import Link from 'next/link'
+import React, { CSSProperties, useMemo } from 'react'
+
+import { useIntl } from '@core/next/intl'
+
+import { colors } from '@condo/domains/common/constants/style'
+import { TicketHint, TicketHintProperty } from '@condo/domains/ticket/utils/clientSchema'
+
+import { TicketHintContent } from './TicketHintContent'
+import { get } from 'lodash'
+
+const ROW_STYLES: CSSProperties = { overflow: 'hidden', backgroundColor: colors.backgroundLightGrey, padding: '20px', borderRadius: '12px' }
+const LINK_STYLES: CSSProperties = { color: 'black', position: 'relative', bottom: '-8px' }
+
+const SMALL_VERTICAL_GUTTER: [Gutter, Gutter] = [0, 10]
```

Comment:
We have a `MAIN_ROW_GUTTER` constant with the same value. Can it be reused in this case?